src/AWS/Signers/V4.elm

```diff
@@ -52,7 +54,7 @@ formatDate date =
     date
         |> formatISO8601
         |> Regex.replace All
-            (regex "([-:\\]|\\.\\d{3})")
+            (regex "[-:]")
```

Comment:
This should do the same thing as

```js
addAuthorization: function addAuthorization(credentials, date) {
    var datetime = AWS.util.date.iso8601(date).replace(/[:\-]|\.\d{3}/g, '');
```

from [v4.js:23](https://github.com/aws/aws-sdk-js/blob/master/lib/signers/v4.js#L23)

Can you provide more info about the runtime error. Maybe it was a bug in the regex, but `[-:]` does not look equivalent to `[:\-]|\.\d{3}`